<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="gradesAVG">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputString">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="InputModul" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="gradesSUM">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="gradesRecommodation">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>startButton()</slot>
  <slot>setValueAVG()</slot>
  <slot>calculate()</slot>
  <slot>grandesAVG()</slot>
 </slots>
</ui>
